tests/test.react-stub-createClass.js

```diff
@@ -0,0 +1,184 @@
+import React, { PropTypes } from 'react';
+import { expect } from 'chai';
+import ReactTestUtils from 'react-addons-test-utils';
+
+import reactStub from 'react-stub';
+
+
+describe('reactStub createClass', function() {
+
+  it('requires a truthy component', function() {
+    expect(() => reactStub()).to.throw(/realComponent is undefined/);
+  });
+
+  it('renders a stub component', function() {
+    var RealComponent = React.createClass({
+      render: function() {
+        return <div/>;
+      },
+    });
+    var Fake = reactStub(RealComponent);
+    // This throws an error if the stub is not a component.
+    ReactTestUtils.renderIntoDocument(<Fake/>);
+  });
+
+  it('lets you find a fake component', function() {
+    var RealComponent = React.createClass({
+      render: function() {
+        return <span>real</span>;
+      },
+    });
+
+    var Fake = reactStub(RealComponent);
+    var container = ReactTestUtils.renderIntoDocument(<Fake/>);
+
+    // This will throw an error if it can't be found.
+    ReactTestUtils.findRenderedComponentWithType(container, Fake);
+  });
+
+  it('lets you add children to the stub', function() {
```

Comment:
I think this test probably isn't necessary because the same stub component will be *returned* in the case of inputting a `createClass` instance or ES6 class.